data/README.md

```diff
@@ -0,0 +1,40 @@
+# Data
+
+## `crash_rates.v1`
+
+The data processing notebook stores data in the following directory:
+
+```
+s3://net-mozaws-prod-us-west-2-pipeline-analysis/amiyaguchi/macrobase/crash_rates/v1
+```
+
+It contains the following fields, as of `v1`.
+
+```
+root
+ |-- timestamp: long (nullable = true)
+ |-- sample_id: string (nullable = true)
+ |-- normalized_channel: string (nullable = true)
+ |-- env_build_version: string (nullable = true)
+ |-- env_build_id: string (nullable = true)
+ |-- app_name: string (nullable = true)
+ |-- os: string (nullable = true)
+ |-- os_version: string (nullable = true)
+ |-- env_build_arch: string (nullable = true)
+ |-- country: string (nullable = true)
+ |-- active_experiment_id: string (nullable = true)
+ |-- active_experiment_branch: string (nullable = true)
```

Comment:
From my understanding declaring experiments this way won't work because a client might have more than one experiment/branch open at any given time. I would talk to @fbertsch about the right way to represent this data (or just leave it out for now)